release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -86,6 +86,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-nodes"]
 === Nodes
 
+==== Graceful Node Shutdown
+
+Graceful Node Shutdown (GNS) enables kubelet to delay forcible eviction of pods in the event of a node shutdown.
+This is important for users who want to ensure that workloads that execute critical operations cannot be interrupted.
+For example, if an operator that flashes network card firmware is shut in a disorderly fashion, hardware can be damaged.
```

Comment:
Suggested rewrite for clarity: Delete this example and add it below alongside the description of the function.